ansible/hosts

```diff
@@ -0,0 +1,2 @@
+[jenkins]
+jenkins
```

Comment:
Because the `inventory-file` is passed in, you shouldn't need this one.

Reply:
I tried that and still couldn't get the ansible playbook to pick up the vars. This was an experiment that I was going to look into more later. For now, I forced it to load the vars in the playbook itself.